packages/ibm-products/src/components/PageHeader/PageHeader.tsx

```diff
@@ -1035,7 +1038,7 @@ export let PageHeader = React.forwardRef(
                             />
                           </>
                         ) : (
-                          widthIsNarrow &&
+                          (widthIsNarrow || pageActions) &&
```

Comment:
@ntkibm This change seems to result in a strange scenario where `thePageActions(...)` renders twice in our case, once in the breadcrumb row (invisibly??) and once more in the title row. This has caused some of our tests to break because they assume that the page actions will only appear in the DOM once. I'm not sure whether to raise a bug for this, it's possible that we are misusing the component. Can you explain in more detail what this change does please?